docs/user/tutorials/01-10-external-authorization-provider.md

```diff
@@ -1,91 +1,80 @@
-# Use an External Authorization Provider to Expose and Secure a Workload
+# Exposing Workloads with oauth2-proxy
 
-This tutorial shows how to expose and secure an HTTPBin Service using an external authorization provider.
+Learn how to configure [oauth2-proxy](https://github.com/oauth2-proxy/manifests/tree/main/helm/oauth2-proxy) external authorization provider in the Istio custom resource (CR).
 
 ## Prerequisites
+* You have the Istio module added. If you use a Kyma domain to expose a workload, also the API Gateway module must be added.
```

Comment:
This prereq is different from the corresponding item in the HP.

Reply:
fixed on HP